products/opensuse/main.pm

```diff
@@ -776,15 +744,16 @@ elsif (get_var("WICKED")) {
     load_wicked_tests();
 }
 elsif (get_var("REGRESSION")) {
-    if (get_var("KEEPHDDS")) {
-        load_login_tests();
-    }
-    else {
+    if (check_var("REGRESSION", "installation")) {
+        load_boot_tests();
         load_inst_tests();
         load_reboot_tests();
+        loadtest "x11regressions/x11regressions_setup";
```

Comment:
if I correctly understand this test is preparation for x11regression test suite , right ? if this is the case why we still have it when we mute all x11regression tests ?

Reply:
x11regressions_setup is preparation for x11regression cases in https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/tests/x11regressions, and these cases will be added into several testsuites as I describe in https://progress.opensuse.org/issues/23516:

Todo list of testsuites:
* regression-installation
* regression-documentation
* regression-firefox
* regression-gnome
* regression-message
* regression-other

Reply:
Sorry haven't read ticket, then it make sense